test/heapwatch/topology-extract-p2p.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
the idea is to have it dependency-free so it can be run on a remote machine with logs